docs/orchestration/agents/fargate.md

```diff
@@ -1,4 +1,12 @@
-# Fargate Agent
+# Fargate Agent (Deprecated)
```

Comment:
Is this _officially_ deprecated or will be in 0.14.0?

Reply:
Will be in 0.14.0. Since we're planning on releasing next week, figured I'd write towards that now.